srcalc/src/main/java/gov/va/med/srcalc/web/controller/admin/ProcedureRowTranslator.java

```diff
@@ -0,0 +1,110 @@
+package gov.va.med.srcalc.web.controller.admin;
+
+import gov.va.med.srcalc.domain.model.Procedure;
+import gov.va.med.srcalc.util.ValidationCodes;
+import gov.va.med.srcalc.util.ValidationUtils2;
+import gov.va.med.srcalc.util.csv.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.apache.commons.csv.CSVRecord;
+
+import com.google.common.base.Optional;
+
+/**
+ * Reads Procedure objects from a Comma-Separated Value (CSV) file.
+ */
+class ProcedureRowTranslator implements RowTranslator<Procedure>
+{
+    private static final String FIRST_HEADER_CELL = "CPT";
+    
+    @Override
+    public boolean isHeaderRow(final CSVRecord record)
+    {
+        return record.get(0).equalsIgnoreCase(FIRST_HEADER_CELL);
```

Comment:
I'm assuming it will be a requirement that the header must be labeled CPT to be a valid header. I could see a user having a header like "CPT code", "code", or just "procedure".